data.sql

```diff
@@ -6,4 +6,20 @@ INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
 
 INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Pikachu', 'January 07, 2021', 1, '0', 15.04);
 
-INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Devimon', 'May 12, 2017', 5, '1', 11);
\ No newline at end of file
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Devimon', 'May 12, 2017', 5, '1', 11);
+
+-- /* Added for Project2 requierements*/
+
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Charmander', 'February 08, 2020', 0, '0', -11);
+
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Plantmon', 'November 15, 2021', 2, '1', -5.7);
+
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Squirtle', 'April 02, 1993', 3, '0', -12.13);
+
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Angemon', 'June 12, 2005', 1, '1', -45);
+
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Boarmon', 'June 07, 2005', 7, '1', 20.4);
+
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Blossom', 'October 13, 1998', 3, '1', 17);
+
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Ditto', 'May 14, 2022', 4, '1', 22);
```

Comment:
- Pleas, kindly use false or true to indicate boleans as this is more readable and maintainable.

Reply:
@zieeco Hi reviewer, if there's nothing else besides that to change, please mark it as approved since that's not in the requirements for this project and the previous reviewers didn't mention anything about this, thank you!

Reply:
@zieeco That should be in the 'OPTIONAL" comments since that's not something that is required obligatory, I'll make the changes anyways since as you say it's a better way, but please mark it as approved.